storage/Utils/AsciiFile.h

```diff
@@ -55,10 +59,15 @@ namespace storage
 	vector<string>& get_lines() { return lines; }
 	const vector<string>& get_lines() const { return lines; }
 
+	void set_lines(const vector<string>& lines) { this->lines = lines; }
+
     protected:
 
+	static const int DEFAULT_PERMISSIONS = 0666;
```

Comment:
please document why this so open permissions are ok as it looks very open.

Reply:
It is the default permissions for any file. The resulting file will have 666 - umask. With 666 we are simply using the default value that the system uses.